src/coordinator/storage/fanout/storage.go

```diff
@@ -117,6 +117,12 @@ func (s *fanoutStorage) Type() storage.Type {
 
 func (s *fanoutStorage) FetchBlocks(
 	ctx context.Context, query *storage.FetchQuery, options *storage.FetchOptions) (block.Result, error) {
+	for _, store := range s.stores {
+		if store.Type() == storage.TypeLocalDC {
+			return store.FetchBlocks(ctx, query, options)
```

Comment:
This may not be correct if we have multiple local storages (I think that may be the case for different resolutions?)

Reply:
just for now until Ben's diff is done and we start supporting remote as well. Will add TODO